### PR TITLE
Harmonize logging string for repositories

### PR DESCRIPTION
**Pull request by pfeuffer**
## Proposed changes

Harmonizes the representation of repositories in log statements. The `toString()` method in the class `Repository` is changed, so that the repository can be used directly instead of `getNamespaceAndName()` or other handmade logs.

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] PR is well described
- [x] Target branch is not master (in most cases develop should bet the target of choice) 
- [x] Code does not conflict with target branch
- [x] Definition of Done's fulfilled: [DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/definition-of-done.md) // [UI DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/ui-dod.md)

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: scm-core/src/main/java/sonia/scm/repository/Repository.java [hunk 1]
```diff
@@ -352,6 +352,10 @@ public int hashCode() {
 
   @Override
   public String toString() {
+    return namespace + "/" + name + " (" + id + ")";
```
Comment (sdorra) [inline scm-core/src/main/java/sonia/scm/repository/Repository.java:355]: `String.format`?

File: scm-core/src/main/java/sonia/scm/repository/RepositoryLog.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package sonia.scm.repository;
+
+public class RepositoryLog {
```
Comment (sdorra) [inline scm-core/src/main/java/sonia/scm/repository/RepositoryLog.java:27]: Do we need this class?

**Review by sdorra — CHANGES_REQUESTED** [review]

File: scm-core/src/main/java/sonia/scm/repository/RepositoryLog.java (hunk 2 above)
Comment (pfeuffer) [reply scm-core/src/main/java/sonia/scm/repository/RepositoryLog.java]: Ups, sorry!

File: scm-core/src/main/java/sonia/scm/repository/Repository.java (hunk 1 above)
Comment (pfeuffer) [reply scm-core/src/main/java/sonia/scm/repository/Repository.java:355]: Second thought was to cache this, but you know ... cache invalidation...

**Pull request merged**